packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -275,13 +275,18 @@ function _Autocomplete<ItemType>(
 
           {!isLoading && items.length === 0 && (
             <div className={cx(styles.item, styles.disabled)}>
-              {noMatchesMessage}
+              <Subheading margin="spacingXs" className="styles.NoMatchesTitle">
```

Comment:
```suggestion
              <Subheading margin="none" className={styles.noMatchesTitle}>
```

We are using camelCase for class names, that's why I rename the class to "noMatchesTitle"
And we are always passing it using styles as an object


I also suggest using `margin="none"` so the message does not have an inconsistent spacing around it
And we only get the spacing coming from the Popover itself